tinyxml2.cpp

```diff
@@ -204,6 +205,7 @@ char* StrPair::ParseText( char* p, const char* endTag, int strFlags )
             return p + length;
         }
         ++p;
+        TIXMLASSERT( p );

```

Comment:
Right, but the only way p could be 0 is if it started at -1, which seems unlikely.